Web/starter_project_barbies/components/team/TeamMemberCard.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { BsFacebook, BsLinkedin, BsInstagram, BsEmojiSmileUpsideDown } from 'react-icons/bs';
+import Image from "next/image";
+import { TeamMember } from "@/types/teams";
+
+export type TeamMemberCardProps = {
+  teamMember: TeamMember
+}
+
+const SocialMediaIcon: {[index: string]:any} = {
+  "facebook": <BsFacebook className="fill-gray-400 w-6 h-6" />,
+  "linkedin": <BsLinkedin className="fill-gray-400 w-6 h-6" />,
+  "instagram": <BsInstagram className="fill-gray-400 w-6 h-6" />,
+  "default": <BsEmojiSmileUpsideDown className="fill-gray-400 w-6 h-6" />
+}
+
+const TeamMemberCard: React.FC<TeamMemberCardProps> = ({
+  teamMember
```

Comment:
you can destruct the teamMember here to avoid having to use teamMember. multiple times